File: dataset/output-bk/MISTRAL_codestral-latest/Task168_MISTRAL_codestral-latest.c
#include <stdio.h>
#include <string.h>

void tweakLetters(char* word, int* tweaks, char* result) {
    for (int i = 0; i < strlen(word); i++) {
        char c = ((word[i] - 'a' + tweaks[i]) % 26) + 'a';
        result[i] = c;
    }
    result[strlen(word)] = '\0';
}

File: dataset/output/MISTRAL_codestral-latest/Task168_MISTRAL_codestral-latest.java
public class Main {
    public static String tweakLetters(String word, int[] tweaks) {
        StringBuilder result = new StringBuilder();
        for (int i = 0; i < word.length(); i++) {
            char c = (char) (((word.charAt(i) - 'a' + tweaks[i]) % 26) + 'a');
            result.append(c);
        }
        return result.toString();
    }
}